### DISCUSSION TITLE
I cannot see the symbol of the project on Ubuntu WSL2

### DISCUSSION THREAD

**Discussion by IamSuperVegeta** [starship/starship#1611] [Q&A] [answered]
Hi,
I installed StarShip on Ubuntu (WSL2) everything looks fine but I cannot see the symbol in my project.

I went through the FAQ and I run the test commands and they work fine. Any help would be appreciated :) 

The example below is a Python project. I have also tried a ruby project and it has the same issue.
As you can see the tests commands work
![image](https://user-images.githubusercontent.com/28528894/90960524-81876800-e4e5-11ea-86d3-70a0cd1d6b2d.png)

**Comment by davidkna** [comment, marked as answer]
Do you have python installed in wsl?  (A python package providing the plain `python` command to be specific, this can be changed to `python2` or `python3` in the config)
I looked at the python module and that one looks like it won't show up if python isn't installed.

For further debugging please try running `STARSHIP_LOG=trace starship prompt` in wsl and posting the output.

**Reply by IamSuperVegeta** [reply]
Thank you ,Once I added a python virtual environment is started working. 
I do have Python3 installed and a requirements.txt , so this is to due with starship looking for python without the version as you mentioned. 

When you say changing python version to be specific do you mean in the starship config ?

**Reply by davidkna** [reply]
Yes. You should be able to put this in the starship config:
```toml
# ~/.config/starship.toml

[python]
python_binary = "python3"
```

**Reply by IamSuperVegeta** [reply]
Thank you very much 😊
